e2e-with-targets/02_main_app.R

```diff
@@ -0,0 +1,110 @@
+## app.R
+
+# loading libraries -------------------------------------------------------------------------------
+# if you don't have {box} package yet, install it with:
+# install.packages("box")
```

Comment:
Do we need this comment? Maybe it's enough to tell that you need to do `renv::restore()` to install all packages to run this?